auth_app/urls.py

```diff
@@ -17,5 +17,6 @@
     ),
     path('register/', Registration.as_view(), name='register'),
 
-    path('profile/<pk>', Profile.as_view(template_name='auth_app/profile.html'), name='profile'),
+    path('profile/<pk>/', Profile.as_view(template_name='auth_app/profile.html'), name='profile'),
```

Comment:
Only a question: why did you change it?

Reply:
URL in Django must have a `/` at the end for a convention.